### DISCUSSION TITLE
Installing only the binaries from LibreOffice?

### DISCUSSION THREAD

**Discussion by gilzow** [pimcore/pimcore#12785] [Q&A / Support] [answered]
Trying to help a client with their Pimcore installation, specifically the [Additional Tools](https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/Additional_Tools_Installation.html#page_Additional-Tools-Installation). The system they are running on is debian-based, but does not have `apt`, `apt-get`, or `dpkg`.  I've been able to get everything else installed with the exception of LibreOffice, and LibreOffice-related packages.

Does anyone know of an alternative way to install LibreOffice? OR does Pimcore support using alternatives like https://github.com/PHPOffice?

**Comment by kingjia90** [comment]
Pimcore requires `https://github.com/PHPOffice/PhpSpreadsheet` but is used only for exporting in xlsx file
For example, see: https://github.com/pimcore/pimcore/pull/12736/files
but it cannot be used as alternative to LibreOffice afaik.

**Comment by gilzow** [comment, marked as answer]
For anyone else that comes across this issue, what I ended up doing was using the [AppImage version of LIbreOffice](https://www.libreoffice.org/download/appimage/): 

- Download the AppImage
- Make the image executable
- Extract the contents via `./LibreOffice-still.basic-x86_64.AppImage --appimage-extract` 
- copy the contents of `./squashfs-root/opt/libreoffice7.2/program/*` to the location where I could store them post-build stage
- Add that location to `$PATH`
